clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestManager.java

```diff
@@ -59,7 +59,7 @@ public interface RequestManager {
      * <em>all</em> request managers is critical to ensure that we can complete as many of the consumer's shutdown
      * tasks as possible within the user-provided timeout.
      */
-    default PollResult pollOnClose() {
+    default PollResult pollOnClose(long currentTimeMs) {
```

Comment:
nit: Since we’ve added a new parameter currentTimeMs: long, I think we should add a description of this new parameter to the JavaDoc of the method. This would also help us align with the pull method.
WYDT ?

Reply:
makes sense, done.